discord/user.py

```diff
@@ -338,6 +342,11 @@ def blocked(self):
         email: str
             The new email you wish to change to.
             Only applicable to user accounts.
+        Optional[:class:`HypeSquadHouse`]
+            If this is supplied also supply ``first_name`` and ``last_name`` fields of str type.
```

Comment:
ditto

Reply:
When i was told about the non user endpoint they told me to include these fields. I am not sure if changing houses would fail houses without them I'm going to go check on this rn

Reply:
Can confirm a POST to `/hypesquad/online` does not require first_name and last_name fields updating PR now